osu.Game.Rulesets.Osu/Mods/OsuModBlindTravel.cs

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Diagnostics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.UI;
+using osu.Framework.Graphics;
+using osuTK;
+using osu.Game.Screens.Play;
+using osu.Framework.Threading;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Scoring;
+using osu.Framework.Bindables;
+using osu.Game.Screens;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Graphics.Containers;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModBlindTravel : ModWithVisibilityAdjustment, IUpdatableByPlayfield, IApplicableToScoreProcessor, IApplicableToPlayer, IApplicableToDrawableRuleset<OsuHitObject>
+
+    {
+        public override string Name => "Blind Travel";
+        public override string Acronym => "BT";
+        public override IconUsage? Icon => FontAwesome.Solid.PlaneDeparture;
+        public override ModType Type => ModType.Fun;
+        public override string Description => "Trust yourself into the void";
+        public override double ScoreMultiplier => 1;
+
+        private const double default_follow_delay = 100;
+
+        [SettingSource("Camera delay", "Milliseconds for the camera to catch up with your cursor")]
+        public BindableDouble CameraDelay { get; } = new BindableDouble(default_follow_delay)
+        {
+            MinValue = default_follow_delay,
+            MaxValue = default_follow_delay * 10,
+            Precision = default_follow_delay,
+        };
+
+        private Double cameraDelay => CameraDelay.Value;
+
+        private float applyIncreasedVisibilityDuration = 1000;
+
+        private float increasedVisibilityDuration = 1000;
+
+        private const float default_scope_scale = 1;
+
+        private const float default_scope_fov = 0.6f;
+
+        [SettingSource("Base FOV", "Adjust the base field of view")]
```

Comment:
I'd probably call it zoom not FOV. FOV is generally used in terms of 3D, not 2D.